qa/vagrant/src/main/java/org/elasticsearch/packaging/test/ArchiveTestCase.java

```diff
@@ -218,7 +218,7 @@ public void test53JavaHomeWithSpecialCharacters() throws Exception {
 
             } finally {
                 //clean up sym link
-                sh.run("cmd /c del /F /Q 'C:\\Program Files (x86)\\java' ");
+                sh.run("cmd /c rmdir 'C:\\Program Files (x86)\\java' ");
```

Comment:
Hopefully it will pass, but I think the problem is the path with spaces (apostrophes didn't help) . Note that del command should pass when run on windows.
If it still fails I would use fileutils.rm